test/api_searchProducts_test.dart

```diff
@@ -98,7 +98,7 @@ void main() {
       expect(result.count > 30000, true);
     });
 
-    test('search products by keywords', () async {
+    /*test('search products by keywords', () async {
```

Comment:
Similar comments about skipping tests, https://pub.dev/packages/test#skipping-tests
Alternatively, we can just completely delete the test

Reply:
Thanks @peterwvj, yeah you have right I will use the skip annotation, it is better and cleaner